src/main/assembly/dist/cfg/application.properties

```diff
@@ -10,17 +10,20 @@ daemon.http.port=20190
 #
 # 1.   Created in the drafts directory.
 # 2.   On submit: copied to stage and then that copy is moved to the submit-to directory.
-# 3a)  If archived successfully the bag in the draft-copy is deleted
+# 3a)  If archived successfully the bag in the stage is moved to submit-to
 # 3b)  If the deposit is rejected, the depositor can reopen the draft, modify it and resubmit it.
 #
-# stage and submit-to must be on a single mount to allow moving
-# stage.upload and draft must be on a single mount to allow moving
+# stage and submit-to must be on a single mount to allow moving from the first to the latter
+# stage.upload and draft must be on a single mount to allow moving zip-content into the draft bag
 #
 deposits.stage.upload=/var/opt/dans.knaw.nl/tmp/easy-deposit-api/stage-upload
 deposits.drafts=/var/opt/dans.knaw.nl/tmp/easy-deposit-api/drafts
 deposits.stage=/var/opt/dans.knaw.nl/tmp/easy-deposit-api/stage
 deposits.submit-to=/var/opt/dans.knaw.nl/tmp/easy-ingest-flow-inbox
 
+# file rights that allow easy-ingest-flow to process deposits from its inbox
```

Comment:
```suggestion
# The group to assign to the files and directories making up the deposit, before moving it to submit-to.
```

Let's keep this self-contained and not bring in easy-ingest-flow in the description.